crates/vm/levm/src/precompiles.rs

```diff
@@ -164,7 +164,12 @@ pub fn ecrecover(
 
     // Parse the input elements, first as a slice of bytes and then as an specific type of the crate
     let hash = calldata.get(0..32).ok_or(InternalError::SlicingError)?;
-    let message = Message::parse_slice(hash).map_err(|_| PrecompileError::ParsingInputError)?;
+    let message = match Message::parse_slice(hash) {
+        Ok(msg) => msg,
+        Err(_) => {
+            return Ok(Bytes::new());
+        }
+    };
```

Comment:
Hi Dami! Good morning.

What do you think of this?
```suggestion
    let Ok(signature) = Signature::parse_standard_slice(sig) else {
        return Ok(Bytes::new());
    };
```

Do you like it? Or do you prefer your version?

Reply:
PS: Honest question! No need to accept it if you don't like!

You have already made "your reviewer" happy ;)

Reply:
Done [aef49b3](https://github.com/lambdaclass/ethrex/pull/1570/commits/aef49b32bedac0594c3ae281fb4eb8a372fe59fa)!